tests/rpc_test.go

```diff
@@ -758,7 +758,7 @@ func TestEth_EstimateGas(t *testing.T) {
 	err := json.Unmarshal(rpcRes.Result, &gas)
 	require.NoError(t, err, string(rpcRes.Result))
 
-	require.Equal(t, "0xef7e", gas)
+	require.Equal(t, "0xf552", gas)
```

Comment:
I wonder why do we have to change this all the time? shouldn't the minimum always be 21,000?

Reply:
yeah that's a good question, I'm not totally sure why it changes